examples/next-login/src/pages/home/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react'
+import LogIn from './login'
+import SignIn from './signup'
+import { SwitchButton, Stack } from '@ultraviolet/ui'
+import styled from '@emotion/styled'
+
+const StyledSwitchButton = styled(SwitchButton)`
+  margin: auto;
+`
+const StyledPage = styled(Stack)`
+  height: 100%;
+  width: 100%;
+`
+const Home = () => {
+  const [tab, setTab] = useState('login')
+  return (
+    <StyledPage>
+      <StyledSwitchButton
+        name="switch button"
+        leftButton={{
+          label: 'Log In',
+          value: 'login',
+        }}
+        rightButton={{
+          label: 'Sign Up',
+          value: 'signup',
+        }}
+        value="login"
+        onChange={event =>
+          setTab((event.currentTarget as HTMLInputElement).value)
+        }
+      />
+      {tab === 'login' ? <LogIn /> : <SignIn />}
```

Comment:
if we use tabs may be it can be good to have a switch, it will be a closer pattern to most routing library :)

```
let tabLoaded = undefined
switch (tab) {
case 'login':
    tabLoaded = <Login/>
    break;
  default:
    tabLoaded = <SignIn />
}

return tabLoaded
```